codevalidator-api/src/main/java/org/sitenv/vocabularies/loader/code/icd10/Icd10CmLoader.java

```diff
@@ -52,7 +52,17 @@ public void load(List<File> filesToLoad) {
 			VocabularyRepository.truncateModel(dbConnection, Icd10CmModel.class);
 			logger.info(dbConnection.getName() + ".Icd10CmModel Datastore Truncated... records remaining: " + VocabularyRepository.getRecordCount(dbConnection, Icd10CmModel.class));
 
+			VocabularyRepository.updateIndexProperties(dbConnection, Icd10CmModel.class, true);
 		
+			String insertQueryPrefix = "insert into " + Icd10CmModel.class.getSimpleName() + " (code, displayName) values ";
+			
+			StrBuilder insertQueryBuilder = new StrBuilder(insertQueryPrefix);
+			insertQueryBuilder.ensureCapacity(1000);
+			
+			int totalCount = 0, pendingCount = 0;
+			
+			dbConnection.declareIntent(new OIntentMassiveInsert());
```

Comment:
As suggested by the [OrientDB performance tuning section](http://orientdb.com/docs/last/Performance-Tuning.html#massive_insertion), tells the database to prepare itself for handling, well, ... a massive insert.